packages/kit/src/core/dev/plugin.js

```diff
@@ -512,6 +512,9 @@ export const svelte = function (svelte_config) {
 			...svelte_config.compilerOptions,
 			hydratable: !!svelte_config.kit.browser.hydrate
 		},
+		hot: {
+			optimistic: false
```

Comment:
rixo mentioned changing the default value in svelte-hmr so this isn't needed after, unless you want to prevent users overriding it

Reply:
yeah, that seems better. will apply https://github.com/sveltejs/kit/pull/5108/files#r884334210 later